nagios/config/wavefront-ingestion.cfg

```diff
@@ -0,0 +1,39 @@
+# Nagios command definitions for host and service metrics to the Wavefront server
```

Comment:
Nagios command definitions for host and service metrics to the Operations for Applications server using an  Operations for Applications API token